openbas-api/src/main/java/io/openbas/migration/V3_44__Add_column_executable_arch.java

```diff
@@ -0,0 +1,20 @@
+package io.openbas.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+@Component
+public class V3_44__Add_column_executable_arch extends BaseJavaMigration {
+
+  @Override
+  public void migrate(Context context) throws Exception {
+    Connection connection = context.getConnection();
+    Statement statement = connection.createStatement();
+    statement.execute("ALTER TABLE payloads ADD executable_arch varchar(255);");
+    statement.execute("UPDATE payloads SET executable_arch = 'x86_64' WHERE payload_type ='Executable';");
+  }
```

Comment:
Maybe you can try use check to verify if payload is executable so executable_arch is not null?

Reply:
Discussed together, I'm not going to make this change. 

For me its better to add this kind of validation on the api/service. I included a validation in `PayloadApi`